libbeat/_meta/config/ssl.reference.yml.tmpl

```diff
@@ -7,6 +7,12 @@
 # matches the names identified within the certificate.
 # * certificate, which verifies that the provided certificate is signed by a
 # trusted authority (CA), but does not perform any hostname verification.
+# * strict, which verifies that the provided certificate is signed by a trusted
+# authority (CA) and also verifies that the server's hostname (or IP address)
+# matches the names identified within the certificate. If the Subject Alternative
+# Name is empty, it returns an error.
```

Comment:
It also means that `CommonName` must not be used for the domain name or ip, right?